src/views/blocks/event-datetime.php

```diff
@@ -46,7 +46,7 @@
 $time_zone_label = $this->attr( 'timeZoneLabel' );
 
 if ( is_null( $time_zone_label ) ) {
-	$time_zone_label = Tribe__Events__Timezones::get_event_timezone_abbr( $event_id );
+	$time_zone_label = Tribe__Events__Timezones::is_mode( 'site' ) ? $time_zone_label = Tribe__Events__Timezones::wp_timezone_abbr( $local_start_time ) : Tribe__Events__Timezones::get_event_timezone_abbr( $event_id );
```

Comment:
you're assigning `$time_zone_label` twice here.

Reply:
I realized this after merging and changed it in this [PR](https://github.com/the-events-calendar/the-events-calendar/pull/3632).